p2p/test/transport/transport_test.go

```diff
@@ -358,6 +378,9 @@ func TestMoreStreamsThanOurLimits(t *testing.T) {
 	const streamCount = 1024
 	for _, tc := range transportsToTest {
 		t.Run(tc.Name, func(t *testing.T) {
+			if strings.Contains(tc.Name, "WebRTC") {
+				t.Skip("This test potentially exhausts the uint16 WebRTC stream ID space.")
```

Comment:
We only have have of that, since client and server use even and odd stream IDs, respectively (or the other way around). That's still a lot, but I've seen this fail on CI on Windows occasionally.